AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DiffuseProbeGridAdded.py

```diff
@@ -0,0 +1,190 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+class Tests:
+    creation_undo = (
+        "UNDO Entity creation success",
+        "UNDO Entity creation failed")
+    creation_redo = (
+        "REDO Entity creation success",
+        "REDO Entity creation failed")
+    diffuse_probe_grid_creation = (
+        "Diffuse Probe Grid Entity successfully created",
+        "Diffuse Probe Grid Entity failed to be created")
+    diffuse_probe_grid_component = (
+        "Entity has a Diffuse Probe Grid component",
+        "Entity failed to find Diffuse Probe Grid component")
+    diffuse_probe_grid_disabled = (
+        "Diffuse Probe Grid component disabled",
+        "Diffuse Probe Grid component was not disabled")
+    box_shape_component = (
+        "Entity has a Box Shape component",
+        "Entity did not have a Box Shape component")
```

Comment:
does this get used or just the for loop test_shape?

Reply:
Ah you're right, that's not necessary anymore. I'll remove.